presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -72,8 +72,8 @@
 
     protected <T> void assertBlock(Block block, Supplier<BlockBuilder> newBlockBuilder, T[] expectedValues)
     {
-        assertBlockSize(block);
-        assertRetainedSize(block);
+//        assertBlockSize(block);
```

Comment:
@yingsu00 Is this code commented out on purpose?

Reply:
> @yingsu00 Is this code commented out on purpose?

@mbasmanova  Thanks for catching this. I updated the PR and made sure there are no other commented out lines.